### PR TITLE
Typo

### PR DESCRIPTION
**Pull request by nadegepepin**

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/lgtm

File: content/en/docs/concepts/services-networking/dns-pod-service.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ search <namespace>.svc.cluster.local svc.cluster.local cluster.local
 options ndots:5
 ```
 
-In summary, a Pod in the _test_ namespace can successfully resolve either 
+In summary, a Pod in the _prod_ namespace can successfully resolve either 
 `data.prod` or `data.prod.svc.cluster.local`.
```
Comment (Sea-n) [inline content/en/docs/concepts/services-networking/dns-pod-service.md:53]: I think "Pod in the prod namespace" is better fit this context.

```suggestion
In summary, a Pod in the _test_ namespace can successfully resolve either 
`data.test` or `data.test.svc.cluster.local`.
```

**Review by Sea-n — COMMENTED** [review]
Nice catch! Your PR makes this logic right.

But I think it's a little confuse for reader since the main article supposed Pod is in the `test` namespace, while the summary supposed Pod is in the `prod` namespace.

**Comment by reylejano** [comment]
I think the existing documentation is actually correct although confusing
Yes the changes in this PR is correct but the original content is meant to illustrate that  a Pod in namespace _a_ can resolve a service innamespace _b_ by referring to namespace _b_ like in `data.b` or `data.b.svc.cluster.local`
/hold

**Comment by nadegepepin** [comment]
Oh. That makes perfect sense. Thanks for clarifying. I probably did not
spend enough time reading though this. Discard the change request.

**Pull request closed without merging**